insights/parsers/ps.py

```diff
@@ -342,3 +341,53 @@ def parse_content(self, content):
     def children(self, ppid):
         """list: Returns a list of dict for all rows with `ppid` as parent PID"""
         return [row for row in self.data if row['PPID'] == ppid]
+
+
+add_filter(Specs.ps_aexww, "COMMAND")
+
+
+@parser(Specs.ps_aexww)
+class PsAexww(Ps):
+    """
+    Class to parse the output of ``ps aexww`` command
+
+    Sample input data::
+
+          PID TTY      STAT   TIME COMMAND
+            1 ?        Ss     0:08 /usr/lib/systemd/systemd --switched-root --system --deserialize 22
+            2 ?        S      0:00 [kthreadd]
+            3 ?        S      0:00 [ksoftirqd/0]
+            5 ?        S<     0:00 [kworker/0:0H]
+            7 ?        S      0:01 [migration/0]
+        24477 ?        Ssl    6:56 /usr/bin/openshift-router PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin HOSTNAME=node1 RELOAD_INTERVAL=10s
+
+    Examples:
+        >>> type(ps_aexww)
+        <class 'insights.parsers.ps.PsAexww'>
+        >>> ps_aexww.get_environ('/usr/bin/openshift-router')['RELOAD_INTERVAL']
+        '10s'
+    """
+    command_name = 'COMMAND'
+    first_column = 'PID'
+    max_splits = 4
+
+    def get_environ(self, proc):
+        """
+        Searches for the first command matching ``proc`` and returns its
+        environment variable as a dict.
+
+        Returns:
+            dict: Dictionary with environment variable name as key and containing  environment variable value
+            ``{}`` if ``proc`` is not found.
+
+        .. note:: 'proc' must contain the command
+        """
+        env = {}
+        for row in self.data:
+            if row[self.command_name].startswith(proc.strip() + ' '):
+                if '=' in row['ARGS']:
```

Comment:
Have you checked https://github.com/RedHatInsights/insights-core/pull/1848#discussion_r276935713